### PR TITLE
Add KodeLife.app v0.4.0

### PR DESCRIPTION
**Pull request by jzgdev**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [*] `brew cask audit --download {{cask_file}}` is error-free.
- [*] `brew cask style --fix {{cask_file}}` reports no offenses.
- [*] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [*] Named the cask according to the [token reference].
- [*] `brew cask install {{cask_file}}` worked successfully.
- [*] `brew cask uninstall {{cask_file}}` worked successfully.
- [*] Checked there are no [open pull requests] for the same cask.
- [*] Checked the cask was not already refused in [closed issues].
- [*] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/kodelife.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'kodelife' do
+  version '0.4.0'
+  sha256 '5aa576d094a514cd9c519db01786fb31321133523ecd23e572412a3aa0cad2f3'
+
+  url 'http://hexler.net/mint/pepper/orderedlist/downloads/download.php?file=http%3A//hexler.net/pub/kodelife/kodelife-0.4.0-osx.zip'
```
Comment (miccal) [inline Casks/kodelife.rb]: Please change this to `url "http://hexler.net/pub/kodelife/kodelife-#{version}-osx.zip"`.

Also, please add an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by miccal — CHANGES_REQUESTED** [review]

**Review by jzgdev — COMMENTED** [review]
Requested changes made

**Comment by miccal** [comment]
I am not sure what you have done, but you have added a bunch of other commits to your PR and have not updated `kodelife.rb` as requested.

**Comment by vitorgalvao** [comment]
@jzgdev Please try again in a new PR. That’ll be easier for everyone.

**Pull request closed without merging**